src/tests/ftest/launch.py

```diff
@@ -1346,7 +1380,14 @@ def process_the_cores(avocado_logs_dir, test_yaml, args):
     if not cores:
         return
 
-    install_debuginfos()
+    try:
+        install_debuginfos()
+    except RuntimeError as error:
+        print(error)
+        print("Removing core files to avoid archiving them")
+        for corefile in cores:
+            os.remove(corefile)
+        exit(1)
```

Comment:
Is `exit(1)` the correct course of action here?  Shouldn't we continue and run the next test, but eventually return `1` to not lose the problem - possibly using a unique internal mask to print an appropriate message at the end.

Reply:
Yeah, probably.  I was just following what the previous behaviour was doing when the debuginfo installation failed, but now that we are handling the situation, pressing on is probably more appropriate.

Reply:
Done.